mappings/dependencies.json

```diff
@@ -788,6 +805,10 @@
         "Xamarin.AndroidX.Slice.Core"
       ]
     },
+    {
+      "id": "Xamarin.AndroidX.Tracing",
```

Comment:
Should this be `Xamarin.AndroidX.Tracing.Tracing`?

Reply:
true. Let me rerun local builds one more time. Not sure why this was not updated.